PROM_THINKING.md

```diff
@@ -0,0 +1,119 @@
+# Bla
```

Comment:
bla? :)

Reply:
This file can be moved aside, perhaps to a google document or something, it shouldn't be part of the committed code.